manager/src/main/resources/application.properties

```diff
@@ -38,4 +38,10 @@ quarkus.oidc.client-id=${EVENT_BRIDGE_SSO_CLIENT_ID}
 %dev.quarkus.oidc.credentials.secret=secret
 
 # Native resources
-quarkus.native.resources.includes=exception/exceptionInfo.json
\ No newline at end of file
+quarkus.native.resources.includes=exception/exceptionInfo.json
+
+# AWS
```

Comment:
See: https://quarkus.io/guides/all-config#quarkus-amazon-iam_quarkus.iam.aws-aws-services-configurations

For local-dev, we can use [localstack](https://github.com/localstack/localstack) to mock the functionality of AWS Secrets Manager and therefore we can simply hard-code the access and secret keyts.